README.md

```diff
@@ -1,16 +1,26 @@
-# Azure Policy
+# Azure Policy Samples
 
-Check here for a current list of [**known issues**](#known-issues).
+This repository contains samples of Azure Policies that can be used as reference for creating and assigning policies to your subscriptions and resource groups. For a full list of samples with descriptions, see [Policy samples](https://docs.microsoft.com/azure/governance/policy/samples/) on docs.microsoft.com.
```

Comment:
Docs doesn't have a full list of what's in this repo.  Only about 50% actually and that will likely change to pattern focused examples instead of RP examples in the future.

Reply:
That is what was previously posted in the readme. I'll remove the claim that a full list of samples exists.